src/lib.rs

```diff
@@ -17,6 +17,8 @@ pub struct Pid<T: FloatCore> {
     pub i_limit: T,
     /// Limit of contribution of D term `(-d_limit <= D <= d_limit)`
     pub d_limit: T,
+    /// Limit of the sum of PID terms `(-limit <= P + I + D <+ limit)`
+    pub limit : T,
```

Comment:
Remember to follow the general rust style guide.
```suggestion
    pub limit: T,
```
`cargo fmt` does a lot for you, so you can also run it to see what it changes rather than looking up style info.